babel.config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   presets: ['@babel/preset-env', '@babel/preset-react'],
-  plugins: ['transform-class-properties'],
+  plugins: ['transform-class-properties', '@babel/plugin-transform-runtime'],
```

Comment:
I do not know why we need this. I just know that without it the app errors out with "regeneratorRuntime is not defined"

Reply:
I think this may contain some further details about the cause: https://github.com/babel/babel/issues/9849